Source/Scene/Camera.js

```diff
@@ -1134,6 +1146,10 @@ define([
     var scratchToHPRRight = new Cartesian3();
 
     function directionUpToHeadingPitchRoll(camera, position, orientation, result) {
+        if (!defined(result)) {
```

Comment:
this if defined should be removed - my mistake - bc this method does not return anything